model.py

```diff
@@ -0,0 +1,199 @@
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.linear_model import LinearRegression
+import warnings
+import pickle
+import logging
+from .ModelEvaluation import ModelEvaluation
+
+warnings.filterwarnings("ignore")
+
+# Set up logging
+logging.basicConfig(
+    filename='models/house_price/logs/model_training.log',
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
```

Comment:
#### Issue:
The current implementation results in an error if the `logs/` directory does not exist. 

#### Recommendation:
You should ensure that the directory is created before attempting to write logs. This can be achieved by either creating the `logs/` directory explicitly or directly creating the entire path for `logs/` folder, as in:
```python
os.makedirs(log_directory, exist_ok=True)
```

This change will prevent runtime errors related to missing directories and improve the robustness of the logging setup.